topics/proteomics/tutorials/metaquantome-data-creation/tutorial.md

```diff
@@ -0,0 +1,640 @@
+ 
+metaQuantome offers differential abundance analysis, principal components analysis, and clustered heat map visualizations, 
+across multiple experimental conditions. metaQuantome, an open source tool, is available via command line and also 
+accessible via Galaxy platform for reproducible analysis. As a first step for metaQuantome analysis, metaproteomics 
+data needs to be made compatible for subsequent analysis. With this in mind, we have developed a metaQuantome data 
+generation workflow tutorial that ill help users generate inputs for metaQuantome analysis.
+
+![Workflow](../../images/metaquantomeworkflow.png){: width="75%"}
+
+To demonstrate the use of the data creation workflow, we have used a thermophilic biogas reactor dataset wherein municipal 
+food waste and manure is digested to generate methane gas. After one round in the reactor, the microbial community was 
+simplified and enriched via serial dilution. This inoculum was then transferred to a solution of cellulose from Norwegian 
+Spruce and incubated at 65°C. Triplicate samples were taken in a time series from 0 to 43 hours after inoculation and mass 
+spectrometry data was acquired on a Q-Exactive (Thermo) mass spectrometer. For this training, we have chosen two time points-8 hour and 33 hour.
+
+![Dataset](../../images/biogasdataset.png){: width="75%"}
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# **Pretreatments**
+
+The first step in a tutorial is to get the data from the zenodo link provided and making sure that it is in the correct format.
+
+## *Get data*
+
+> ### {% icon hands_on %} Hands-on: Data upload
+>
+> 1. Create a new history for this tutorial
+> 2. Import the files ( 6 MZML files, a Protein FASTA file and Experimental Design file) from [Zenodo]({{ page.zenodo_link }}) 
+>    or from the shared data library (`GTN - Material` -> `{{ page.topic_name }}` -> `{{ page.title }}`)
+>
+>    ```
+>    https://zenodo.org/record/4037137/files/ExperimentalDesign.tsv
+>    https://zenodo.org/record/4037137/files/ProteinDB_cRAP.fasta
+>    https://zenodo.org/record/4037137/files/T2_A1.mzml
+>    https://zenodo.org/record/4037137/files/T2_B1.mzml
+>    https://zenodo.org/record/4037137/files/T7A_1.mzml
+>    https://zenodo.org/record/4037137/files/T7B_1.mzml
+>    ```
+>    {% include snippets/import_via_link.md %}
+>    {% include snippets/import_from_data_library.md %}
+>
+>
+> 3. Build a **Dataset list** for the four mzml files.
+>    - Click the **Operations on multiple datasets** check box at the top of the history panel
+>       ![Operations on multiple datasets button](../../images/operations_icon.png)
+>    - Check the four boxes next to the mzml files.
+>    - Click **For all selected...** and choose **Build dataset list**
+>
+> 4. Rename the datasets (IF needed)
+> 5. Check that the datatype ( Make sure they are in the correct formats).
+>
+>    {% include snippets/change_datatype.md datatype="datatypes" %}
+>
```

Comment:
I suggest moving the step for checking datatypes to before the step for building a dataset list. That way, the files are in the correct format from the start.